lib/main_ltp.pm

```diff
@@ -41,6 +42,10 @@ sub load_kernel_tests {
         if (get_var('FLAVOR', '') =~ /Incidents-Kernel/) {
             loadtest_kernel 'update_kernel';
         }
+        if (is_alp && is_qemu) {
```

Comment:
I know we don't have this combination yet, but `is_qemu` is also applicable to the ISO, right?